test/sax/sax_test.rb

```diff
@@ -395,17 +387,15 @@ def test_sax_open_close_element
                    [:start_element, :mid],
                    [:text, ""],
                    [:end_element, :mid],
-                   [:end_element, :top]
-                  ])
+                   [:end_element, :top]])
   end
 
   def test_sax_empty_element
     Ox::default_options = $ox_sax_options
     parse_compare(%{  <top>  </top>},
                   [[:start_element, :top],
                    [:text, " "],
-                   [:end_element, :top]
-                  ], AllSax, { :skip => :skip_white })
+                   [:end_element, :top]], AllSax, { :skip => :skip_white })
```

Comment:
I prefer the original. It's easier to read and easier to edit if new items are to be added from the list.